packages/server/lib/utils/utils.ts

```diff
@@ -145,17 +145,19 @@ export function parseJsonDateAware(input: string) {
 
 export function parseConnectionConfigParamsFromTemplate(template: ProviderTemplate): string[] {
```

Comment:
`parseConnectionConfigParamsFromTemplate` is only used in `config.controller.ts` so it is fine changing its logic